ppl/src/main/java/org/opensearch/sql/ppl/parser/AstExpressionBuilder.java

```diff
@@ -812,4 +784,136 @@ public UnresolvedExpression visitNumericSpanValue(
   public UnresolvedExpression visitLogWithBaseSpan(OpenSearchPPLParser.LogWithBaseSpanContext ctx) {
     return org.opensearch.sql.ast.dsl.AstDSL.stringLiteral(ctx.getText());
   }
+
+  // Visitor methods for search expressions
+  @Override
+  public SearchExpression visitGroupedExpression(OpenSearchPPLParser.GroupedExpressionContext ctx) {
```

Comment:
Is better to create SearchExpressionAnalyzer instead?

SearchExpression is only used by search command, and SearchExpression is not composite with other expression.

Reply:
You mean SearchExpressionBuilder?